FirebaseMLModelDownloader/Sources/ModelFileManager.swift

```diff
@@ -15,7 +15,7 @@
 import Foundation
 
 /// Manager for common file operations.
-enum ModelFileManager {
+class ModelFileManager: FileManager {
```

Comment:
Did you mean to include this here as well?

Reply:
Ah good catch! That was by accident, but I was wondering if I should mock the FileManager dependency (so that I can pass a different directory and still test that file operations work). Do you think that may be worth it? If not, I'll change this back to enum.